oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/service/ServiceTraffic.java

```diff
@@ -98,8 +121,11 @@ public void deserialize(final RemoteData remoteData) {
         public ServiceTraffic storage2Entity(final Map<String, Object> dbMap) {
             ServiceTraffic serviceTraffic = new ServiceTraffic();
             serviceTraffic.setName((String) dbMap.get(NAME));
-            serviceTraffic.setNodeType(NodeType.valueOf(((Number) dbMap.get(NODE_TYPE)).intValue()));
+            serviceTraffic.setShortName((String) dbMap.get(SHORT_NAME));
             serviceTraffic.setGroup((String) dbMap.get(GROUP));
+            if (dbMap.get(LAYER) != null) {
+                serviceTraffic.setLayer(Layer.valueOf(((Number) dbMap.get(LAYER)).intValue()));
+            }
```

Comment:
You should add a `else` here to set Layer as default(`undefined`) for the legacy data.